build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ls -op packer | grep -v / | awk '{print $8}' | tail -n +2 | while read image
+
+do
+    packer build packer/$image
+done
```

Comment:
Okay. my ride back home leaves soon as I just found out so I have to be OoO until monday, but I will work on the other tasks at hand until then.